### DISCUSSION TITLE
Using ONLY $mod in bindsym --release for opening rofi or applications in general

### DISCUSSION THREAD

**Discussion by Doktertje** [i3/i3#4674] [Q&A] [answered]
Dear i3wm community,

I have a question about using a keybind with ONLY `$mod` (here: Super key), instead of `$mod + <another key>`.
The goal is to be able to open `rofi` using a single keystroke, similar to opening the Start menu on Windows.

It seems that opening `rofi` using the following binding does NOT work:

`bindsym $mod --release exec --no-startup-id -rofi -show run -font "Source Code Pro 12`

...while using:

`bindsym --release Super_L exec --no-startup-id -rofi -show run -font "Source Code Pro 12`

does work.

Does anyone know why i3wm does not allow using only `$mod`, while it does work/recognise `Super_L` (which is the same key on my keyboard)?

Please let me know if (and how) I should provide additional details.

TIA!

**Comment by splpaul** [comment]
Hello
Besides config file readability why aren't you just using the --release one? As far as I understand, it has the same behavior and if you are worried you or someone is gonna have trouble reading the config file you could just add a comment.

**Reply by Doktertje** [reply]
Hi @splpaul,

Thank you for your comment and suggestion.
I've modified my code example to indicate the fact that using `--release` with `$mod` also does not work (which is what I meant to show as well).

The point of my question is really that `i3` does not seem to understand that `$mod` can be used by itself (as far as I know the documentation does not explicitly state that `bindsym` requires two keys).

You are very right that using `Super_L --release` would solve the issue, but I can imagine that this problem exists for other modifier keys as well, and I am really looking for a universal solution.

To me, it seems odd that `$mod` is not recognised, but `Super_L` is, and while using `$mod` is the preferred/generalised method of binding keys in i3

**Reply by Airblader** [reply]
Modifier keys are called that because they _modify_ something. If you look at the [keybinding docs](https://i3wm.org/docs/userguide.html#keybindings), the syntax is

```
bindsym [--release] [<Group>+][<Modifiers>+]<keysym> command
```

i.e. the keysym is mandatory, the modifiers are optional.

**Reply by Doktertje** [reply]
Dear @Airblader,

Thank you for your clarification and also for your valuable work on `i3`/`i3-gaps`.
It is now clear that this behaviour on Windows is different from how `i3` handles keybinding.
My question has been fully answered.

Thank you again!

**Comment by Doktertje** [comment, marked as answer]
This question has been answered by the contribution of @Airblader in this thread!
(Written separately here as I cannot mark his comment as an answer)

Modifier keys are called that because they _modify_ something. If you look at the [keybinding docs](https://i3wm.org/docs/userguide.html#keybindings), the syntax is

`bindsym [--release] [<Group>+][<Modifiers>+]<keysym> command`

i.e. the keysym is mandatory, the modifiers are optional.

**Reply by Airblader** [reply]
Note that there are tools that allow you to bind to modifier keys directly. Just in i3 we do not support that.
